convert.go

```diff
@@ -320,6 +320,29 @@ func unquoteIfPossible(s string) (string, error) {
 	return strconv.Unquote(s)
 }
 
+func wrapParagraph(s string, l int, prefix string) string {
```

Comment:
I think we should just replace the function with

```go
func wrapParagraph(s string, l int, prefix string) string {
	ps := strings.Split(s, "\n\n")

	for i := range ps {
		ps[i] = wrapText(ps[i], l, prefix)
	}

	return strings.Join(ps, "\n\n"+prefix)
}
```

Which is essentially the same code since TrimSpace is already called in wrapText.